### DISCUSSION TITLE
Changing Badge Text?

### DISCUSSION THREAD

**Discussion by JustinBack** [tosdr/CrispCMS#43] [Q&A]
I think the Badge Text is not clear enough on what it represents.

![](https://beta.tosdr.org/api/badge/service/duckduckgo)
![](https://beta.tosdr.org/api/badge/service/tos;dr)
![](https://beta.tosdr.org/api/badge/service/whatsapp)
![](https://beta.tosdr.org/api/badge/service/paypal)
![](https://beta.tosdr.org/api/badge/service/facebook)

Should this Text Be Adjusted to for instance "Privacy Grade A" or something? Otherwise it is not that clear on what tosdr.org does

Maybe @michielbdejong, @Peepo1 or @DrKJeff16 has some ideas?

**Comment by DrKJeff16** [comment]
Since we don't only deal with privacy, Grade A/B... is okay for me :P
